.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+<< A description of my changes >>
+
+See #<< ISSUE >>.
+
+---
+
+Checklist:
+
+ - [ ] I have split up this change into one or more appropriately-delineated commits.
+ - [ ] The first line of each commit is of the form "<component>: do something"
```

Comment:
The angle brackets on this line cause the word 'component' not to render (since it considers them as an attempt to use HTML tags)